cmd/release/prepare/runner.go

```diff
@@ -0,0 +1,89 @@
+package prepare
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+)
+
+const (
+	ChangelogFile = "CHANGELOG.md"
+	VersionFile   = "pkg/project/project.go"
+)
+
+func runReleaseError(cmd *cobra.Command, args []string) error {
+	absoluteCurrentFolder, _ := filepath.Abs(filepath.Dir(os.Args[0]))
+	repositoryName := fmt.Sprintf("%s/%s", filepath.Base(filepath.Dir(absoluteCurrentFolder)), filepath.Base(absoluteCurrentFolder))
+
+	currentVersion, err := replaceVersionInFile(VersionFile)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = addReleaseToChangelog(time.Now().Format("2006-01-02"), currentVersion, repositoryName)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func replaceVersionInFile(file string) (string, error) {
+	// Read contents of the file containing current version
+	filecontents, err := ioutil.ReadFile(file)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	versionFileContents := string(filecontents)
+
+	versionRegex := regexp.MustCompile(`version\s*=\s*"([0-9]+\.[0-9]+\.[0-9]+)-dev"`)
+	currentVersion := versionRegex.FindSubmatch(filecontents)
+	if len(currentVersion) < 1 {
+		return "", microerror.Maskf(executionFailedError, "No version was found")
```

Comment:
Error annotations should start with lower letter:

```suggestion
		return "", microerror.Maskf(executionFailedError, "no version was found")
```